locations/spiders/dsv_locker_za.py

```diff
@@ -0,0 +1,30 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class DSVLockerZASpider(Spider):
+    name = "dsv_locker_za"
+    item_attributes = {"brand": "DSV Locker", "brand_wikidata": "Q1155771"}
+    allowed_domains = ["dsv-clientzone-map.afrigis.co.za"]
+    start_urls = ["https://dsv-clientzone-map.afrigis.co.za/proxy/api/wfs/?layer=locker"]
+
+    def parse(self, response):
+        for data in response.json()["result"]["features"]:
+            data = data["properties"]
+            item = DictParser.parse(data)
+
+            item["name"] = data.get("location_name")
+            item["street"] = data.get("suburb")
```

Comment:
This doesn't look correct. In one example, `data["suburb"] = "Northmead"` and `data["street_address"] = 2, Fifth Avenue, Northmead, 1501, Johannesburg, Gauteng`.

Therefore "Fifth Avenue" is the street, and "Northmead" is the "city".

I see the source data has `suburb`, `town` and `province` but ATP only allows `city` and `state`. `city` in ATP should be the smallest administrative region corresponding to a suburb (or town if the town has no suburbs). `state` in ATP should be the 2nd level of government beneath the federal government. Perhaps one level in the source data needs to be skipped?

Reply:
So, I guess suburb should be skipped.

Reply:
Actually I think you'd skip Johannesburg in this case, and keep Northmead ("city") and Gauteng ("state"). The ATP term of "city" isn't what the name implies sadly. It generally means "suburb, or town/city if the town/city has no suburbs".

Reply:
Yep I think that's right, just set `item["city"] = data["suburb"]`